Tryouts/Prototypes/Shell/Node-launcher/Lib/src/Launcher.ts

```diff
@@ -0,0 +1,39 @@
+import { execFile } from 'child_process';
+import { WindowConfig } from './WindowConfig';
+
+export class Launcher {
+    private argsArray: string[] = [];
+
+    constructor(_config?: WindowConfig ) { 
+
+    }
+
+    private processArgs(_config?: WindowConfig) {
```

Comment:
I'm confused about naming conventions here. Why the underscore before the function parameter?